npm/qsharp/ux/circuit-vis/events.ts

```diff
@@ -397,19 +398,23 @@ class CircuitEvents {
       !removeQubitLineButton.hasAttribute("data-event-added")
     ) {
       removeQubitLineButton.addEventListener("click", () => {
-        const check = (op: Operation) => {
-          if (op.targets.some((reg) => reg.qubit == this.qubits.length - 1)) {
+        const check = (comp: Component) => {
+          const targets =
+            comp.type === "Measurement" ? comp.results : comp.targets;
+          if (targets.some((reg) => reg.qubit == this.qubits.length - 1)) {
             return true;
           }
+          const controls =
+            comp.type === "Measurement" ? comp.qubits : comp.controls;
           if (
-            op.controls &&
-            op.controls.some((reg) => reg.qubit == this.qubits.length - 1)
+            controls &&
+            controls.some((reg) => reg.qubit == this.qubits.length - 1)
```

Comment:
So the issue here is that `controls` and `targets` got renamed for measurement? I don't think this looks bad - it may even be more readable since I don't know what a "control" and "target" is for a measurement.

Reply:
Yep, those were confusing terms for measurements, especially given that they worked the other way around from how you'd expect them to from non-measurements.